.gitignore

```diff
@@ -15,4 +15,3 @@
 /libpeerconnection.log
 npm-debug.log
 testem.log
-.idea
```

Comment:
We should probably keep this and remove the existing .idea directory - no reason for my IDE settings to be part of the git repo.